src/Commands/sailonLagoonAssets/Lagoon/cli.dockerfile

```diff
@@ -42,6 +44,23 @@ RUN if [ -f "package.json" ]; then \
     npm run build; \
   fi
 
+#######################################################
```

Comment:
This seems like a change unrelated to the PR's stated aim?

Could you explain what's happening here, if you don't mind - that is, why does this live in the dockerfile, rather than post-rollout?

Reply:
ARRG - yes it is :D - I had it forked and didn't realize. Will redo tomorrow! Will close this PR